cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,19 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    const userName = 'Alex1205@gmail.com';
+    const password = 'kassandra01';
 
+    cy.visit('https://react-redux.realworld.io');
+
+    cy.get(':nth-child(2) > .nav-link')
+      .click();
+    cy.get(':nth-child(1) > .form-control')
+      .type(userName);
```

Comment:
Replace the input userName with email when calling the type() method, since email is the correct input for the email address input field.